codegen/gradle.properties

```diff
@@ -1,2 +1,2 @@
-smithyVersion=1.28.1
+smithyVersion=1.30.0
```

Comment:
I think it'd be helpful to mention in the description that this is bumping the Smithy version to 1.30.0.